### PR TITLE
Update windows-10-1903-removed-features.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3777
Added NOTE.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/deployment/planning/windows-10-1903-removed-features.md [hunk 1]
```diff
@@ -26,7 +26,10 @@ The following features and functionalities are removed from the installed produc
...
-| Desktop messaging app doesn't offer messages sync |                                                                                                                          The messaging app on Desktop has a sync feature that can be used to sync SMS text messages received from Windows Mobile and keep a copy of them on the Desktop. The sync feature has been removed from all devices. Due to this change, you will only be able to access messages from the device that received the message.                                                                                                                          |
+| Desktop messaging app doesn't offer messages sync |                                                                                                                          The messaging app on Desktop has a sync feature that can be used to sync SMS text messages received from Windows Mobile and keep a copy of them on the Desktop. The sync feature has been removed from all devices. Due to this change, you will only be able to access messages from the device that received the message.
```
Comment (JohanFreelancer9) [inline windows/deployment/planning/windows-10-1903-removed-features.md]:
```suggestion
| Desktop messaging app doesn't offer messages sync |                                                                                                                          The Messaging app on desktop has a sync feature that can be used to sync SMS text messages received from Windows Mobile and keep a copy of them on the desktop. The sync feature has been removed from all devices. Due to this change, you will only be able to access messages from the device that received the message. 
```

File: windows/deployment/planning/windows-10-1903-removed-features.md [hunk 2]
```diff
@@ -26,7 +26,10 @@ The following features and functionalities are removed from the installed produc
...
-| Desktop messaging app doesn't offer messages sync |                                                                                                                          The messaging app on Desktop has a sync feature that can be used to sync SMS text messages received from Windows Mobile and keep a copy of them on the Desktop. The sync feature has been removed from all devices. Due to this change, you will only be able to access messages from the device that received the message.                                                                                                                          |
+| Desktop Messaging app doesn't offer messages sync |                                                                                                                          The Messaging app on desktop has a sync feature that can be used to sync SMS text messages received from Windows Mobile and keep a copy of them on the desktop. The sync feature has been removed from all devices. Due to this change, you will only be able to access messages from the device that received the message.
```
Comment (nenonix) [inline windows/deployment/planning/windows-10-1903-removed-features.md]:
```suggestion
| Desktop Messaging app doesn't offer messages sync |                                                                                                                          The messaging app on desktop has a sync feature that can be used to sync SMS text messages received from Windows Mobile and keep a copy of them on the desktop. The sync feature has been removed from all devices. Due to this change, you will only be able to access messages from the device that received the message. 
```

Comment (nenonix) [inline windows/deployment/planning/windows-10-1903-removed-features.md]:
```suggestion
| Desktop messaging app doesn't offer messages sync |                                                                                                                          The Messaging app on desktop has a sync feature that can be used to sync SMS text messages received from Windows Mobile and keep a copy of them on the desktop. The sync feature has been removed from all devices. Due to this change, you will only be able to access messages from the device that received the message. 
```

File: windows/deployment/planning/windows-10-1903-removed-features.md [hunk 3]
```diff
@@ -26,7 +26,10 @@ The following features and functionalities are removed from the installed produc
...
+| Desktop Messaging app doesn't offer messages sync |                                                                                                                          The Messaging app on desktop has a sync feature that can be used to sync SMS text messages received from Windows Mobile and keep a copy of them on the desktop. The sync feature has been removed from all devices. Due to this change, you will only be able to access messages from the device that received the message. 
+
+> [!NOTE] 
+> This only relates to messages from Windows Phone devices in the Messaging app. It does NOT apply to messages from Android devices in the Your Phone app.
```
Comment (nenonix) [inline windows/deployment/planning/windows-10-1903-removed-features.md]:
```suggestion
> This only relates to messages from Windows Phone devices in the Messaging app. It does NOT apply to messages from Android devices in the Your Phone app.
```
```suggestion
> This only relates to messages from Windows Phone devices in the messaging app. It does **not** apply to messages from Android devices in the Your Phone app.
```

Comment (JohanFreelancer9) [reply windows/deployment/planning/windows-10-1903-removed-features.md]: @nenonix  Hi Nicole. "Messaging" in "Messaging app" is capitalized in other Microsoft docs (see https://www.microsoft.com/en-us/p/microsoft-messaging/9wzdncrfjbq6?activetab=pivot:overviewtab)?

Comment (Reeced40) [reply windows/deployment/planning/windows-10-1903-removed-features.md]: I have capitalized in the latest commits.

**Comment by e0i** [comment]
@greg-lindsay The PR that addresses issue #3777 is available for review. Thanks.

**Comment by e0i** [comment]
@greg-lindsay Following up for updates.

**Comment by e0i** [comment]
@greg-lindsay Following up for updates.
cc @dansimp

**Comment by greg-lindsay** [comment]
Sorry but we can't accept changes to this page. All edits even small ones need to be processed through legal due to the sensitive nature of deprecation.

**Pull request closed without merging**